Update collections, crypto, certs, and asn1 tag

etcwilde opened the pull request:
Updating tags for swift-collections, swift-crypto, swift-certificates, and swift-asn1 to pick up changes needed for FreeBSD.

etcwilde committed:
Update collections, crypto, certs, and asn1 tag

Updating tags for swift-collections, swift-crypto, swift-certificates,
and swift-asn1 to pick up changes needed for FreeBSD.

etcwilde commented:
@swift-ci please test

etcwilde commented:
@swift-ci please test

shahmishal commented on utils/update_checkout/update-checkout-config.json:
```diff
@@ -139,10 +139,10 @@
-                "swift-crypto": "3.0.0",
-                "swift-certificates": "1.0.1",
-                "swift-asn1": "1.0.0",
+                "swift-collections": "1.1.5",
+                "swift-crypto": "3.12.3",
+                "swift-certificates": "1.10.0",
+                "swift-asn1": "1.3.2",
```
Please also update the `rebranch` and `next` scheme.

etcwilde commented:
`error WIX0103: Cannot find the File file 'T:\\Program Files\\Swift\\Platforms\Windows.platform\Developer\SDKs\Windows.sdk\usr\lib\swift\windows\_FoundationCollections.swiftmodule\aarch64.swiftdoc'`

I need to update the swift installer scripts repo with the new module layout.

etcwilde commented:
https://github.com/swiftlang/swift-installer-scripts/pull/434
@swift-ci please test Windows

etcwilde committed:
build.ps1: Update reference to CCryptoBoringSSL

The CCryptoBoringSSL library changed from C to C++, which means that the
static archive on Windows changed names, from `CCryptoBoringSSL.lib` to
`libCCryptoBoringSSL.lib`. The SourceKit-LSP SwiftPM test invocation
explicitly passed `CCryptoBoringSSL.lib` to the SwiftPM invocation,
resulting in the wrong search path. Pointing it at the correct location
now.

etcwilde commented:
https://github.com/swiftlang/swift-installer-scripts/pull/434
@swift-ci please test Windows

etcwilde commented:
@swift-ci please test Linux
